app/views/layouts/application.html.erb

```diff
@@ -40,9 +40,16 @@
     <% end %>
 
     <% if flash[:alert] %>
-      <%= render "components/error_alert", {
-        message: flash[:alert]
-      } %>
+      <% if flash[:alert].is_a? Enumerable %>
+        <%= render "govuk_publishing_components/components/error_summary", {
+          title: "TODO: Update gem to not require title",
```

Comment:
According to https://trello.com/c/H09wIghW, we should [add the title "Something has gone wrong"](https://docs.google.com/document/d/15euIDtRWAClra8y4VX1ZMSnXqS3SN24fJof6EcGRdWQ/edit#heading=h.u0b4qt5inox0).